addons/sale_timesheet/models/project.py

```diff
@@ -299,7 +299,7 @@ def action_billable_time_button(self):
             'views': [
                 [self.env.ref('hr_timesheet.timesheet_view_tree_user').id, 'tree'],
                 [self.env.ref('hr_timesheet.view_kanban_account_analytic_line').id, 'kanban'],
-                [self.env.ref('sale_timesheet.view_hr_timesheet_line_pivot_billing_rate').id, 'pivot'],
+                [self.env.ref('sale_timesheet.timesheets_analysis_report_pivot_inherit').id, 'pivot'],
```

Comment:
This action displays the `account.analytic.line` model, so I think we could either removing this view in that action or give another pivot view for AAL model since the pivot that you have given is for the `Timesheet Analysis Report`. 🙂